posts/a-primer-on-snowflake-stored-procedures.md

```diff
@@ -34,28 +88,28 @@ y = stored_procedure1(x) --NOT ALLOWED
   * There are indirect ways to use the returned value of a stored procedure:
     * You can call a stored procedure inside a stored procedure and use the JavaScript from the outer procedure to retrieve and store the output of the inner. *The outer stored procedure is still unable to return more than one value to its caller.*
     * You can call the stored procedure, then call the RESULT_SCAN function and pass it the statement ID generated for the procedure.
-    * If the stored procedure is a [caller’s rights stored procedure](###caller's-rights-stored-procedures), you can store a result set in a temporary (or permanent) table, and use the table after returning from the stored procedure call.
+    * If the stored procedure is a [caller’s rights stored procedure](####caller's-rights-stored-procedures), you can store a result set in a temporary (or permanent) table, and use the table after returning from the stored procedure call.
```

Comment:
I'm not sure if this hyperlink to the other section is working. But that could be just because it isn't in the blog form.